Gemfile

```diff
@@ -1,8 +1,13 @@
 source "https://rubygems.org"
 
+gemspec
+
 git "https://github.com/refinery/refinerycms", branch: "master" do
-  gem "refinerycms-core"
-  gem "refinerycms-testing"
+  gem "refinerycms"
```

Comment:
why does it need all of `refinerycms`?

Reply:
I was unable to go to the admin without the all `refinerycms`

Reply:
what I mean is that everything is designed to work with just `refinerycms-core` as every other extension attaches to that. We shouldn't need the whole package just to see the admin

Reply:
Yes i know but i had multiple bugs with only `refinerycms-core`. I could rollback this change but we should merge the others changes.

What do you think?

Reply:
I think it's ultimately fine, I just wanted to provide the commentary.